src/mame/drivers/p2000t.cpp

```diff
@@ -250,7 +351,27 @@ void p2000t_state::p2000t(machine_config &config)
 	MDCR(config, m_mdcr, 0);
 
 	/* internal ram */
-	RAM(config, m_ram).set_default_size("16K").set_extra_options("16K,32K,48K,64K,80K,102K");
+	RAM(config, m_ram).set_default_size("48K").set_extra_options("16K,32K,48K,64K,80K,102K");
```

Comment:
Why have you changed the default here?  It seems the base system configuration was used as the default before.